solarnet/roles/ipfs/templates/config.j2

```diff
@@ -51,30 +51,14 @@
     "Writable": false,
     "HTTPHeaders": {
       "Access-Control-Allow-Origin": [
-        "https://gateway.ipfs.io",
-        "https://ipfs.io",
-        "http://gateway.ipfs.io",
-        "http://ipfs.io",
-        "http://h.ipfs.io",
-        "http://localhost",
-        "http://localhost:8080",
-        "http://127.0.0.1",
-        "http://127.0.0.1:8080"
+        "*"
       ]
     }
   },
   "API": {
     "HTTPHeaders": {
       "Access-Control-Allow-Origin": [
-        "https://gateway.ipfs.io",
-        "https://ipfs.io",
-        "http://gateway.ipfs.io",
-        "http://ipfs.io",
-        "http://h.ipfs.io",
-        "http://localhost",
-        "http://localhost:8080",
-        "http://127.0.0.1",
-        "http://127.0.0.1:8080"
```

Comment:
we should only change this for the `Gateway`, not the API -- actually, this should NOT have the public gateways, and only have the default parameters.

now it relies on one level of isolation: docker not rebinding the port. if it rebinds the port, then the api is usable outside.